zboss/include/osif/build_info.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#ifndef ZBOSS_BUILD_INFO_H__
+#define ZBOSS_BUILD_INFO_H__
+
+/**
+ * This is an autogenerated file.
+ * Please use release scripts to update it's contents.
+ *
+ * Repositories:
+ * platform_ncs       d513afa25bc093fc51790b6d262355e8a424fce9 https://bitbucket-zoi.dsr-corporation.com/scm/ZOI/platform_ncs.git
+ * zboss-stack-src    2f74919ccf140ce7e3fbd974d055ae5c2b88fe50 https://bitbucket-zoi.dsr-corporation.com/scm/zoi/zboss_stable
+ * fw-nrfconnect-nrf  c04978740f2261af2a0c9b1148597c73bd2a9967 https://github.com/maciekfabia/fw-nrfconnect-nrf
```

Comment:
Are you releasing this from a developers fork ?
This SHA will change when merged to master, and if @maciekfabia branch is later deleted, it may no longer be possible to created an identical build.

Reply:
I think we have a chicken and egg problem here. I will try to cut this dependency in build projects.

Reply:
Notes regarding this issue:
1. We need Kconfig options to build the ZBOSS stack.
2. Kconfig is provided by Zephyr's build system.
3. There is no option to run Zephyr's build system without building an application.
4. In order to link ZBOSS with an application, the OSIF layer has to be implemented (Zigbee subsys placed inside fw-nrfconnect-nrf repository).
5. If there is an API change in ZBOSS, the Zigbee subsys has to be aligned, thus we cannot use master branch.

As a result - if we want to change ZBOSS API, without having this dependency in build_info header, we have to build an application and ZBOSS, but do not link them together or try to compile the Zigbee subsys.

Zigbee subsys is included when CONFIG_ZIGBEE is enabled, which is required to have the Kconfig options, required by the stack build.

After a lot of testing I found out that to achieve smooth API changes we need to:
 - Make it possible to disable building and linking ZIGBEE library 
 - Make it possible to remove ZBOSS from application links (either through APP_LINK_WITH or CMake commands).

For this PR I will simply bring back ZBOSS header to a state compatible with current fw-nrfconnect-nrf repository state.